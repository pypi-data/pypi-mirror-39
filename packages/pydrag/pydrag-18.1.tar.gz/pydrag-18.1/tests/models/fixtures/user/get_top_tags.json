{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&format=json&limit=50&method=user.getTopTags&user=rj"
            },
            "response": {
                "body": {
                    "string": "{\"toptags\":{\"tag\":[{\"name\":\"rock\",\"count\":\"20\",\"url\":\"https://www.last.fm/tag/rock\"},{\"name\":\"jazz\",\"count\":\"17\",\"url\":\"https://www.last.fm/tag/jazz\"},{\"name\":\"metal\",\"count\":\"10\",\"url\":\"https://www.last.fm/tag/metal\"},{\"name\":\"soul\",\"count\":\"8\",\"url\":\"https://www.last.fm/tag/soul\"},{\"name\":\"female vocalists\",\"count\":\"8\",\"url\":\"https://www.last.fm/tag/female+vocalists\"},{\"name\":\"80s\",\"count\":\"7\",\"url\":\"https://www.last.fm/tag/80s\"},{\"name\":\"funk\",\"count\":\"7\",\"url\":\"https://www.last.fm/tag/funk\"},{\"name\":\"guitar\",\"count\":\"7\",\"url\":\"https://www.last.fm/tag/guitar\"},{\"name\":\"Mellow\",\"count\":\"7\",\"url\":\"https://www.last.fm/tag/Mellow\"},{\"name\":\"singer-songwriter\",\"count\":\"5\",\"url\":\"https://www.last.fm/tag/singer-songwriter\"},{\"name\":\"acoustic\",\"count\":\"5\",\"url\":\"https://www.last.fm/tag/acoustic\"},{\"name\":\"blues\",\"count\":\"5\",\"url\":\"https://www.last.fm/tag/blues\"},{\"name\":\"funky\",\"count\":\"5\",\"url\":\"https://www.last.fm/tag/funky\"},{\"name\":\"checkitout\",\"count\":\"5\",\"url\":\"https://www.last.fm/tag/checkitout\"},{\"name\":\"Stoner Rock\",\"count\":\"4\",\"url\":\"https://www.last.fm/tag/Stoner+Rock\"},{\"name\":\"Hip-Hop\",\"count\":\"3\",\"url\":\"https://www.last.fm/tag/Hip-Hop\"},{\"name\":\"chillout\",\"count\":\"3\",\"url\":\"https://www.last.fm/tag/chillout\"},{\"name\":\"electronic\",\"count\":\"3\",\"url\":\"https://www.last.fm/tag/electronic\"},{\"name\":\"drone rock\",\"count\":\"3\",\"url\":\"https://www.last.fm/tag/drone+rock\"},{\"name\":\"classic rock\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/classic+rock\"},{\"name\":\"country\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/country\"},{\"name\":\"instrumental\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/instrumental\"},{\"name\":\"folk\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/folk\"},{\"name\":\"experimental\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/experimental\"},{\"name\":\"dance\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/dance\"},{\"name\":\"easy listening\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/easy+listening\"},{\"name\":\"psychedelic\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/psychedelic\"},{\"name\":\"bass\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/bass\"},{\"name\":\"covers\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/covers\"},{\"name\":\"Smooth Jazz\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/Smooth+Jazz\"},{\"name\":\"vacuum tube\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/vacuum+tube\"},{\"name\":\"melodic rock\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/melodic+rock\"},{\"name\":\"futurejam\",\"count\":\"2\",\"url\":\"https://www.last.fm/tag/futurejam\"},{\"name\":\"Disco\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/Disco\"},{\"name\":\"indie\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/indie\"},{\"name\":\"pop\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/pop\"},{\"name\":\"japanese\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/japanese\"},{\"name\":\"reggae\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/reggae\"},{\"name\":\"heavy\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/heavy\"},{\"name\":\"ambient\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/ambient\"},{\"name\":\"hard rock\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/hard+rock\"},{\"name\":\"downtempo\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/downtempo\"},{\"name\":\"Progressive rock\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/Progressive+rock\"},{\"name\":\"electro\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/electro\"},{\"name\":\"slow\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/slow\"},{\"name\":\"new wave\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/new+wave\"},{\"name\":\"britpop\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/britpop\"},{\"name\":\"hair metal\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/hair+metal\"},{\"name\":\"soft rock\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/soft+rock\"},{\"name\":\"romantic\",\"count\":\"1\",\"url\":\"https://www.last.fm/tag/romantic\"}],\"@attr\":{\"user\":\"RJ\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "3694"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:28:14 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
