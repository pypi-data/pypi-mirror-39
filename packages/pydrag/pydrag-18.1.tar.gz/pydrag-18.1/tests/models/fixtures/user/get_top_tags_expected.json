{
    "data": [
        {
            "count": 20,
            "name": "rock",
            "url": "https://www.last.fm/tag/rock"
        },
        {
            "count": 17,
            "name": "jazz",
            "url": "https://www.last.fm/tag/jazz"
        },
        {
            "count": 10,
            "name": "metal",
            "url": "https://www.last.fm/tag/metal"
        },
        {
            "count": 8,
            "name": "soul",
            "url": "https://www.last.fm/tag/soul"
        },
        {
            "count": 8,
            "name": "female vocalists",
            "url": "https://www.last.fm/tag/female+vocalists"
        },
        {
            "count": 7,
            "name": "80s",
            "url": "https://www.last.fm/tag/80s"
        },
        {
            "count": 7,
            "name": "funk",
            "url": "https://www.last.fm/tag/funk"
        },
        {
            "count": 7,
            "name": "guitar",
            "url": "https://www.last.fm/tag/guitar"
        },
        {
            "count": 7,
            "name": "Mellow",
            "url": "https://www.last.fm/tag/Mellow"
        },
        {
            "count": 5,
            "name": "singer-songwriter",
            "url": "https://www.last.fm/tag/singer-songwriter"
        },
        {
            "count": 5,
            "name": "acoustic",
            "url": "https://www.last.fm/tag/acoustic"
        },
        {
            "count": 5,
            "name": "blues",
            "url": "https://www.last.fm/tag/blues"
        },
        {
            "count": 5,
            "name": "funky",
            "url": "https://www.last.fm/tag/funky"
        },
        {
            "count": 5,
            "name": "checkitout",
            "url": "https://www.last.fm/tag/checkitout"
        },
        {
            "count": 4,
            "name": "Stoner Rock",
            "url": "https://www.last.fm/tag/Stoner+Rock"
        },
        {
            "count": 3,
            "name": "Hip-Hop",
            "url": "https://www.last.fm/tag/Hip-Hop"
        },
        {
            "count": 3,
            "name": "chillout",
            "url": "https://www.last.fm/tag/chillout"
        },
        {
            "count": 3,
            "name": "electronic",
            "url": "https://www.last.fm/tag/electronic"
        },
        {
            "count": 3,
            "name": "drone rock",
            "url": "https://www.last.fm/tag/drone+rock"
        },
        {
            "count": 2,
            "name": "classic rock",
            "url": "https://www.last.fm/tag/classic+rock"
        },
        {
            "count": 2,
            "name": "country",
            "url": "https://www.last.fm/tag/country"
        },
        {
            "count": 2,
            "name": "instrumental",
            "url": "https://www.last.fm/tag/instrumental"
        },
        {
            "count": 2,
            "name": "folk",
            "url": "https://www.last.fm/tag/folk"
        },
        {
            "count": 2,
            "name": "experimental",
            "url": "https://www.last.fm/tag/experimental"
        },
        {
            "count": 2,
            "name": "dance",
            "url": "https://www.last.fm/tag/dance"
        },
        {
            "count": 2,
            "name": "easy listening",
            "url": "https://www.last.fm/tag/easy+listening"
        },
        {
            "count": 2,
            "name": "psychedelic",
            "url": "https://www.last.fm/tag/psychedelic"
        },
        {
            "count": 2,
            "name": "bass",
            "url": "https://www.last.fm/tag/bass"
        },
        {
            "count": 2,
            "name": "covers",
            "url": "https://www.last.fm/tag/covers"
        },
        {
            "count": 2,
            "name": "Smooth Jazz",
            "url": "https://www.last.fm/tag/Smooth+Jazz"
        },
        {
            "count": 2,
            "name": "vacuum tube",
            "url": "https://www.last.fm/tag/vacuum+tube"
        },
        {
            "count": 2,
            "name": "melodic rock",
            "url": "https://www.last.fm/tag/melodic+rock"
        },
        {
            "count": 2,
            "name": "futurejam",
            "url": "https://www.last.fm/tag/futurejam"
        },
        {
            "count": 1,
            "name": "Disco",
            "url": "https://www.last.fm/tag/Disco"
        },
        {
            "count": 1,
            "name": "indie",
            "url": "https://www.last.fm/tag/indie"
        },
        {
            "count": 1,
            "name": "pop",
            "url": "https://www.last.fm/tag/pop"
        },
        {
            "count": 1,
            "name": "japanese",
            "url": "https://www.last.fm/tag/japanese"
        },
        {
            "count": 1,
            "name": "reggae",
            "url": "https://www.last.fm/tag/reggae"
        },
        {
            "count": 1,
            "name": "heavy",
            "url": "https://www.last.fm/tag/heavy"
        },
        {
            "count": 1,
            "name": "ambient",
            "url": "https://www.last.fm/tag/ambient"
        },
        {
            "count": 1,
            "name": "hard rock",
            "url": "https://www.last.fm/tag/hard+rock"
        },
        {
            "count": 1,
            "name": "downtempo",
            "url": "https://www.last.fm/tag/downtempo"
        },
        {
            "count": 1,
            "name": "Progressive rock",
            "url": "https://www.last.fm/tag/Progressive+rock"
        },
        {
            "count": 1,
            "name": "electro",
            "url": "https://www.last.fm/tag/electro"
        },
        {
            "count": 1,
            "name": "slow",
            "url": "https://www.last.fm/tag/slow"
        },
        {
            "count": 1,
            "name": "new wave",
            "url": "https://www.last.fm/tag/new+wave"
        },
        {
            "count": 1,
            "name": "britpop",
            "url": "https://www.last.fm/tag/britpop"
        },
        {
            "count": 1,
            "name": "hair metal",
            "url": "https://www.last.fm/tag/hair+metal"
        },
        {
            "count": 1,
            "name": "soft rock",
            "url": "https://www.last.fm/tag/soft+rock"
        },
        {
            "count": 1,
            "name": "romantic",
            "url": "https://www.last.fm/tag/romantic"
        }
    ],
    "user": "RJ"
}
